homeassistant/components/nmbs/sensor.py

```diff
@@ -99,7 +100,7 @@ def __init__(self, api_client, live_station):
     @property
     def name(self):
         """Return the sensor default name."""
-        return "NMBS Live"
+        return "{} Live".format(self._name)
```

Comment:
IMHO, if a user defines a name, it should not be appended or prepended with anything.

Right now, "Live" is added. Even if the user didn't want it to.

Reply:
I understand your concern but I am not sure how to differentiate the live from the normal sensor without appending something to one of them.
E.g.
normal sensor: sensor.name
live sensor: sensor.live_name

Reply:
Maybe make name optional, check here if it is set. If it is set just return that name, if not set just return the station concatenated with live?

```python
def name(self):
	if (self._name)
		return self._name
	else
		return "{} Live".format(self._station)
```

Reply:
I would suggest a slightly different version:
```python
def name(self):
	if (self._name is not DEFAULT_NAME)
		return self._name
	else
		return "{} Live".format(self._station)
```
Since the name variable has a default value which we would have to incorporate as well.
What do you think?

Reply:
Sure

Reply:
Sure

Reply:
Great! I will implement the changes and test them asap.

Reply:
The current state of this PR handles this problem. However, I can absolutely understand the concern @frenck raised that a name should be exactly what the user specified it to be. Still I cannot think of another way to deal with this without appending or prepending some a string to identify the live sensor.

Reply:
Just add a unique ID so that the user can change the name from the UI via the entity registry instead of relying on configuration.yaml .

Reply:
That sounds like a perfect solution, thank you @balloob for the tip! I will implement a unique_id asap.